### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Salwa99**
# Bookstore: styling
### Howdy :cowboy_hat_face: 

**In this PR I:**

1. Add the missing JSX elements.
2. Add Style to the project
3. Fix linters errors
4. Follow GitFlow

_Cheers_

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,231 @@
...
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.main-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 5%;
+  padding: 3%;
+  border: 1px solid grey;
+}
```
Comment (emmiiorji) [inline src/App.css]: Kindly ensure that the containing element of the book cards is as given by the design guide. Check the illustration below and take a look at the margins and padding. Also consider adjusting the gap between the cards.

Your design:
image

Design guide:
image

File: src/components/Book.js [hunk 2]
```diff
@@ -43,6 +44,21 @@ function Book({
...
         </div>
+        <div>
+          <CircelProgress percentage={generateRandom} />
+          <div>
+            <h2>
+              {generateRandom}
+              <span>%</span>
+            </h2>
+            <p>Completed</p>
+          </div>
```
Comment (emmiiorji) [inline src/components/Book.js]: Kindly adjust the display of the circular progress and the "completed" text to be horizontally aligned as given on the design guide.

Your design:
image

Figma design:
image

File: src/components/Header.js [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
-    <ul>
-      <li><Link to="/">Books</Link></li>
-      <li><Link to="/categories">Categories</Link></li>
-    </ul>
+    <nav className="navbar">
+      <div className="logo-links">
+        <h1 className="logo">bookstore CMS</h1>
+        <ul>
+          <li className="link1"><Link to="/">BOOKS</Link></li>
+          <li className="link2"><Link to="/categories">CATEGORIES</Link></li>
```
Comment (emmiiorji) [inline src/components/Header.js]: Well done for implementing the navigation between the pages. Kindly ensure that the color of the navigation text changes to the black color.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Salwa99, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of gitflow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salwa99** [comment]
Thank you @emmiiorji :smiley: I'll apply those required changes

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Salwa99,

Good job implementing the changes asked by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ The app is nicely styled 👍🏽
✅ Good communication with the previous reviewer 👍🏽
✅ Add and remove features work fine as expected 👍🏽

## Suggested Changes 

- [ ] It would be nice if you could use the exact blue color given in the original design. It was not stated by the previous reviewer and I highly recommend you please work on that 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**